src/components/publicApi/LoginScreen.tsx

```diff
@@ -30,6 +34,7 @@ interface Props {
   // Options passed to the core login methods:
   accountOptions: EdgeAccountOptions
 
+  onComplete: OnComplete
```

Comment:
Add a comment. Maybe even a JavaDoc comment.

Reply:
```
  /**
   * Called when the user navigates back passed the initialRoute if it was set.
   */
```

Does this seem correct? There appears to be more cases when `onComplete` is called, but I'm not familiar with those cases.